src/coreclr/jit/hwintrinsic.h

```diff
@@ -175,6 +176,18 @@ enum HWIntrinsicFlag : unsigned int
 
     // The intrinsic needs consecutive registers
     HW_Flag_NeedsConsecutiveRegisters = 0x4000,
+
+    // The intrinsic uses scalable registers
+    HW_Flag_Scalable = 0x8000,
+
+    // Returns Per-Element Mask
+    // the intrinsic returns a vector containing elements that are either "all bits set" or "all bits clear"
+    // this output can be used as a per-element mask
+    HW_Flag_ReturnsPerElementMask = 0x10000,
```

Comment:
I am not sure if I follow the definition here or may be remove `"all bits set" or "all bits clear"` because I think we can also use it for "some bits set"?

Reply:
It's `AllBitsSet` or `Zero` on a per-element basis, that is every element is either `AllBitsSet` or `Zero` and no other value.

Reply:
got it